wiki/Contests/VMC/1/en.md

```diff
@@ -0,0 +1,125 @@
+---
+tags:
+  - VMC1
+  - VMC
+---
+
+# Vocaloid Mapping Contest
+
+![VMC banner](https://cdn.discordapp.com/attachments/1092234806479298680/1096272980310495272/banner_e.png)
```

Comment:
put the banner image as `Contests/VMC/1/img/banner.png`, not hotlinked like this.

```suggestion
![VMC banner](img/banner.png)
```